java8/src/main/java/Client/Main.java

```diff
@@ -0,0 +1,62 @@
+package Client;
+
+import Data.Car;
+
+import java.util.*;
+
+import static Client.LogicalOperations.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //1
+        Car c1 = new Car("Mercedes", true, 4);

```

Comment:
naming is important, please use at least firstCar, secondCar, (very rare cases of numbers in variable names)